lib/openstax_kramdown.rb

```diff
@@ -0,0 +1,65 @@
+# https://www.snip2code.com/Snippet/49311/Monkey-patch-for-Kramdown-to-rewrite-rel
```

Comment:
Is this url right, it redirects somewhere else for me

Reply:
It probably wasn't really relevant since the previous version of this file subclassed the Kramdown parser, and this version subclasses the Html parser